core/kazoo_apps/src/kapps_call.erl

```diff
@@ -665,10 +685,17 @@ callee_id_number(#kapps_call{callee_id_number=CIDNumber}) -> CIDNumber.
 set_request(Request, #kapps_call{}=Call) when is_binary(Request) ->
     [RequestUser, RequestRealm] = binary:split(Request, <<"@">>),
     Call#kapps_call{request=Request
-                     ,request_user=knm_converters:normalize(RequestUser)
+                     ,request_user=to_e164(RequestUser)
                      ,request_realm=RequestRealm
                     }.
 
+-ifdef(TEST).
+to_e164(Number) -> Number.
+-else.
+to_e164(Number) ->
+    knm_converters:normalize(Number).
+-endif.
```

Comment:
at least with the wnm_util version, it queried the db to get the regex. We aren't testing to_e164 so skip that check